cmd/gitService.go

```diff
@@ -0,0 +1,119 @@
+package cmd
+
+import (
+	_ "embed"
+	"os"
+	"os/exec"
+
+	log "github.com/sirupsen/logrus"
+)
+
+var DEPLOY_CONFIG_GIT_REPO string = "https://automotivemastermind@dev.azure.com/automotivemastermind/aM/_git/am.devops.deploy"
+var DEPLOY_CONFIG_GIT_REPO_BRANCH string = "local"
+
+var HELM_CONFIG_GIT_REPO string = "https://automotivemastermind@dev.azure.com/automotivemastermind/aM/_git/am.devops.helm"
+var HELM_CONFIG_GIT_REPO_BRANCH string = "local"
```

Comment:
```suggestion
var (
DEPLOY_CONFIG_GIT_REPO = "https://automotivemastermind@dev.azure.com/automotivemastermind/aM/_git/am.devops.deploy"
DEPLOY_CONFIG_GIT_REPO_BRANCH = "local"
HELM_CONFIG_GIT_REPO = "https://automotivemastermind@dev.azure.com/automotivemastermind/aM/_git/am.devops.helm" HELM_CONFIG_GIT_REPO_BRANCH = "local"
)
```

Reply:
also eventually these Uris should be in a configuration file. that way they can be changed without a rebuild of the tool =]

Reply:
I'll do the conversion to a config.json file now